### ISSUE TITLE
Incompatibility between GeoNetwork 3.10 / RNDT: Associated resources

### ISSUE THREAD

**Issue by pdorio** [metadata101/iso19139.rndt#16]
The problem could be related to issue #10

1. Compiling Online Resources via RNDT plugin

![2021-11-04_15h25_25](https://user-images.githubusercontent.com/3297563/140332409-aa207bc6-cdcb-4613-b78b-5c6aee2ebb7e.png)

2. Open the online resources management in the sidebar "Associated resources". The protocol field is empty. Maybe The problem could be related to issue #15

![2021-11-04_15h26_11](https://user-images.githubusercontent.com/3297563/140331734-a20d03e7-5c54-49ca-9aa8-8cc343107d39.png)

3. If you try to delete the online resource from this screen, you get an error

![2021-11-04_15h28_18](https://user-images.githubusercontent.com/3297563/140332056-d9a60a05-61f5-4c87-875e-ea050e99373c.png)

- [ ] Problem 1: The fields of OnlineResources, edited in "Associated resources" are not correctly filled.
- [ ] Problem 2: Problem by deleting

**Comment by silvia-starnini** [comment]
We got similar problems on creating, associating, deleting, updating resources from side panel.

We have overcome some of these issues by copying from the base iso plugin the following files from /process folder: sibling-add, sibling-remove, fcats-add, fcats-remove, source-add, source-remove.
We also modified onlinesrc-add and onlinesrc-remove, but  still we have problems on some kind of resources.
